frontend/src/components/RecommendationBox.tsx

```
@@ -0,0 +1,41 @@
+  const { t } = useI18n()
+
+  return (
+    <motion.div className="bg-white rounded-2xl shadow-md p-6 max-w-2xl text-center border border-gray-200 flex flex-col items-center">
+      <motion.p className="text-gray-700 text-base leading-relaxed mb-6 whitespace-pre-line">
+        {text || t('diagnostic.recommendations.not_found')}
+      </motion.p>
+
+      <button
+        onClick={onToggleFullText}
+        className="mb-4 px-4 py-2 h-10 bg-gray-200 text-black rounded-full text-sm font-medium hover:bg-gray-300 transition-colors min-w-[120px]"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)